src/SAWScript/Proof.hs

```diff
@@ -1072,6 +1076,9 @@ data Evidence
     --   sequent calculus axiom, which connects a hypothesis to a conclusion.
   | AxiomEvidence
 
+  -- | TODO: Add some evidence
```

Comment:
This TODO aside, it would be worth including some documentation about how `MrSolverEvidence` can be produced.